docs/contribute.rst

```diff
@@ -35,6 +35,7 @@ It is challenging to be both concise and descriptive, but that is
 what a well-written summary should do. Consider the commit message
 as something that will be pasted into release notes:
 
+* Maximum line length is 88 for code and 72 for docs.
```

Comment:
Why not make both 88?

Reply:
Because that would go against the [pep8 recommendations](https://peps.python.org/pep-0008/#maximum-line-length). And I agree with them that from the readability point of view this is a good maximum.